myning/chapters/enter_research_facility.py

```diff
@@ -1,15 +1,15 @@
 from myning.config import RESEARCH
 from myning.objects.player import Player
+from myning.objects.research_facility import ResearchFacility
 from myning.utils.file_manager import FileManager
 from myning.utils.io import pick
 from myning.utils.ui import columnate, get_research_string
 
 
 def play():
     while True:
-        player = Player()
-        facility = player.research_facility
-        player.research_facility.check_in()
+        facility = ResearchFacility()
+        facility.check_in()
```

Comment:
since we're checking in in the dashboard, this might be redundant

Reply:
True. Did the migration work this time?